### DISCUSSION TITLE
How do I change the repository website link?

### DISCUSSION THREAD

**Discussion by madebyisaacr** [community/community#46986] [Repositories] [answered]
### Select Topic Area

Question

### Body

On my repository page, there's a link shown under the repository name. It was set automatically by the Vercel integration, but I now have my site hosted on a domain and I want to change the link here to the new domain.

![Screenshot_20230210-103740~2](https://user-images.githubusercontent.com/119639439/218147702-5df87302-53bb-4083-a10d-320bb47ef283.png)

I looked through the repository settings and I can't find a setting for this link. Where do I find the setting to change it?

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @resoluteconquest 

Access the repository through the desktop, or activate the `Desktop mode` option on the mobile phone, then click on this gear icon to change the URL.

![image](https://user-images.githubusercontent.com/47532703/218149380-8d8c06b3-b970-47bd-9867-45d67eb26c35.png)

**Reply by SilasRodrigues19** [reply]
If you want to delete the deployment made by Vercel, you access this link below and click on the trash icon.

https://github.com/resoluteconquest/notiongrid/environments

![image](https://user-images.githubusercontent.com/47532703/218149985-b1040e0f-4348-439a-b411-9e2d2d8be6ae.png)

**Reply by madebyisaacr** [reply]
Thanks Silas!
